### PR TITLE
MIDRC-390 fix separate fields

### PR DESCRIPTION
**Pull request by mfshao**
Jira Ticket: [MIDRC-390](https://ctds-planx.atlassian.net/browse/MIDRC-390)

See also https://github.com/uc-cdis/data-portal/pull/1404

### New Features
- Add a new Guppy Explorer config field `filters.tabs.asTextAggFields` to explicitly specify the filter field names that to be casted from numeric to text 

### Bug Fixes
- Fixed a bug that casts all numeric field in filter into text fields

### Improvements
- Update ES6 image for local dev helper to support running on Apple Silicon Macs

[MIDRC-390]: https://ctds-planx.atlassian.net/browse/MIDRC-390?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ocshawn — DISMISSED** [review]
looks good, might want to add a test for the new functionality

File: stories/conf.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ export const filterConfig = {
       'project',
       'study',
     ],
+    asTextAggFields: [
+      'consortium_id',
```
Comment (ocshawn) [inline stories/conf.js:9]: this looks to break the storybook filters

Comment (mfshao) [reply stories/conf.js:9]: did you regenerate test data?

Comment (ocshawn) [reply stories/conf.js:9]: yes i did, was a clean install locally

Comment (mfshao) [reply stories/conf.js:9]: weird... my storybook looks fine 🤔 
Screenshot 2023-09-15 at 9 13 57 AM

try re-run `sh ./generate_data.sh` ?

Comment (ocshawn) [reply stories/conf.js:9]: figured it out, sorry will get back to testing, on ubuntu need to run the script with `./generate_data.sh` no sh as it does not run correctly :facepalm"

**Review by ocshawn — APPROVED** [review]

**Pull request merged**